
# Map Details
map_size: 32
map_resolution: 1.0
coverage_size: 10.0

obstacles:
 - { id: 1, x: -15.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 2, x: -15.0,  y: -14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 3, x: -15.0,  y: -13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 4, x: -15.0,  y: -12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 5, x: -15.0,  y: -11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 6, x: -15.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 7, x: -15.0,  y: -9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 8, x: -15.0,  y: -8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 9, x: -15.0,  y: -7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 10, x: -15.0,  y: -6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 11, x: -15.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 12, x: -15.0,  y: -4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 13, x: -15.0,  y: -3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 14, x: -15.0,  y: -2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 15, x: -15.0,  y: -1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 16, x: -15.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 17, x: -15.0,  y: 1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 18, x: -15.0,  y: 2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 19, x: -15.0,  y: 3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 20, x: -15.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 21, x: -15.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 22, x: -15.0,  y: 6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 23, x: -15.0,  y: 7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 24, x: -15.0,  y: 8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 25, x: -15.0,  y: 9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 26, x: -15.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 27, x: -15.0,  y: 11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 28, x: -15.0,  y: 12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 29, x: -15.0,  y: 13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 30, x: -15.0,  y: 14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 31, x: -15.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 32, x: -15.0,  y: 16.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 33, x: -14.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 34, x: -14.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 35, x: -14.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 36, x: -13.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 37, x: -13.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 38, x: -13.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 39, x: -12.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 40, x: -12.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 41, x: -12.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 42, x: -11.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 43, x: -11.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 44, x: -11.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 45, x: -10.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 46, x: -10.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 47, x: -10.0,  y: -9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 48, x: -10.0,  y: -8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 49, x: -10.0,  y: -7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 50, x: -10.0,  y: -6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 51, x: -10.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 52, x: -10.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 53, x: -10.0,  y: 1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 54, x: -10.0,  y: 2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 55, x: -10.0,  y: 3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 56, x: -10.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 57, x: -10.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 58, x: -10.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 59, x: -10.0,  y: 11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 60, x: -10.0,  y: 12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 61, x: -10.0,  y: 13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 62, x: -10.0,  y: 14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 63, x: -10.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 64, x: -9.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 65, x: -9.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 66, x: -8.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 67, x: -8.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 68, x: -7.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 69, x: -7.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 70, x: -6.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 71, x: -6.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 72, x: -5.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 73, x: -5.0,  y: -14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 74, x: -5.0,  y: -13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 75, x: -5.0,  y: -12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 76, x: -5.0,  y: -11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 77, x: -5.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 78, x: -5.0,  y: -9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 79, x: -5.0,  y: -8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 80, x: -5.0,  y: -7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 81, x: -5.0,  y: -6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 82, x: -5.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 83, x: -5.0,  y: -4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 84, x: -5.0,  y: -3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 85, x: -5.0,  y: -2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 86, x: -5.0,  y: -1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 87, x: -5.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 88, x: -5.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 89, x: -5.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 90, x: -5.0,  y: 11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 91, x: -5.0,  y: 12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 92, x: -5.0,  y: 13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 93, x: -5.0,  y: 14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 94, x: -5.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 95, x: -5.0,  y: 16.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 96, x: -4.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 97, x: -4.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 98, x: -4.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 99, x: -3.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 100, x: -3.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 101, x: -3.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 102, x: -2.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 103, x: -2.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 104, x: -2.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 105, x: -1.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 106, x: -1.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 107, x: -1.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 108, x: 0.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 109, x: 0.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 110, x: 0.0,  y: -9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 111, x: 0.0,  y: -8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 112, x: 0.0,  y: -7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 113, x: 0.0,  y: -6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 114, x: 0.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 115, x: 0.0,  y: -4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 123, x: 0.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 124, x: 0.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 125, x: 0.0,  y: 6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 126, x: 0.0,  y: 7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 127, x: 0.0,  y: 8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 128, x: 0.0,  y: 9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 129, x: 0.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 130, x: 0.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 131, x: 1.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 132, x: 1.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 133, x: 2.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 134, x: 2.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 135, x: 3.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 136, x: 3.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 137, x: 4.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 138, x: 4.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 139, x: 5.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 140, x: 5.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 141, x: 5.0,  y: -9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 142, x: 5.0,  y: -8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 143, x: 5.0,  y: -7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 144, x: 5.0,  y: -6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 145, x: 5.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 146, x: 5.0,  y: -4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 147, x: 5.0,  y: -3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 148, x: 5.0,  y: -2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 149, x: 5.0,  y: -1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 150, x: 5.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 151, x: 5.0,  y: 1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 152, x: 5.0,  y: 2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 153, x: 5.0,  y: 3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 154, x: 5.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 155, x: 5.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 156, x: 5.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 157, x: 5.0,  y: 11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 158, x: 5.0,  y: 12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 159, x: 5.0,  y: 13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 160, x: 5.0,  y: 14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 161, x: 5.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 162, x: 5.0,  y: 16.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 163, x: 6.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 164, x: 6.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 165, x: 6.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 166, x: 6.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 167, x: 7.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 168, x: 7.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 169, x: 7.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 170, x: 7.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 171, x: 8.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 172, x: 8.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 173, x: 8.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 174, x: 8.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 175, x: 9.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 176, x: 9.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 177, x: 9.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 178, x: 9.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 179, x: 10.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 180, x: 10.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 181, x: 10.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 182, x: 10.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 183, x: 10.0,  y: 1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 184, x: 10.0,  y: 2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 185, x: 10.0,  y: 3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 186, x: 10.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 187, x: 10.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 188, x: 10.0,  y: 6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 189, x: 10.0,  y: 7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 190, x: 10.0,  y: 8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 191, x: 10.0,  y: 9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 192, x: 10.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 193, x: 10.0,  y: 11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 194, x: 10.0,  y: 12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 195, x: 10.0,  y: 13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 196, x: 10.0,  y: 14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 197, x: 10.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 198, x: 11.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 199, x: 11.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 200, x: 11.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 201, x: 12.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 202, x: 12.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 203, x: 12.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 204, x: 13.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 205, x: 13.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 206, x: 13.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 207, x: 14.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 208, x: 14.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 209, x: 14.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 210, x: 15.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 211, x: 15.0,  y: -14.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 212, x: 15.0,  y: -13.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 213, x: 15.0,  y: -12.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 214, x: 15.0,  y: -11.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 215, x: 15.0,  y: -10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 216, x: 15.0,  y: -5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 217, x: 15.0,  y: -4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 218, x: 15.0,  y: -3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 219, x: 15.0,  y: -2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 220, x: 15.0,  y: -1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 221, x: 15.0,  y: 0.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 222, x: 15.0,  y: 1.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 223, x: 15.0,  y: 2.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 224, x: 15.0,  y: 3.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 225, x: 15.0,  y: 4.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 226, x: 15.0,  y: 5.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 227, x: 15.0,  y: 6.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 228, x: 15.0,  y: 7.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 229, x: 15.0,  y: 8.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 230, x: 15.0,  y: 9.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 231, x: 15.0,  y: 10.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 232, x: 15.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 233, x: 16.0,  y: -15.0,  width: 1.0, length: 1.0, height: 20.0 }
 - { id: 234, x: 16.0,  y: 15.0,  width: 1.0, length: 1.0, height: 20.0 }

# Location of the basestation in the map (Will be used to spawn basestation and robots around it)
basestation_x: 0
basestation_y: 0
basestation_z: 0

# Robot parameters
vehicle_num: $(arg num_robots)
nodes_per_robot: 10.0

# Sim Parameters
del_t: 0.5  # timestep between sim updates
